odk1-src/central-submissions.rst

```diff
@@ -3,7 +3,68 @@
+
+Finally, ODK Central offers Public Access Links. A Public Access Link grants anybody in possession of the link the ability to submit to a Form on your server. You can control whether each respondent can submit more than once, and revoke access from any Link at any time.
+
+Submissions sent to Central are available to browse in a preview table, to connect directly to data analysis tools, and for download.
+
+.. _central-submissions-direct:
+
+Direct Web Browser Submissions
+------------------------------
+
+Web Users who are Administrators, Project Managers, or Data Collectors can directly fill Forms in the web browser from the Central administration website. This functionality is provided by Enketo, which does not always behave quite the same as Collect, or support the same features.
+
+   .. image:: /img/central-submissions/new.png
+
+Administrators and Project Managers can begin a survey by going to the :guilabel:`Submissions` tab of the Form, and clicking on the :guilabel:`New` button next to the Submissions header. This will open a new tab which will load the Form in Enketo.
+
+   .. image:: /img/central-submissions/data-collector-form-listing.png
+
+Data Collectors do not have access to the detailed Form management pages. Instead, they will find a :guilabel:`Fill Form` button next to the Form name in the list of Forms on the Project Overview page.
+
+.. _central-submissions-public-link:
+
+Public Access Links
+-------------------
+
+ODK Central allows the distribution of surveys to a broad or open respondent group using Public Access Links. These Links take recipients directly to the Form in their web browser, administered by Enketo.
+
+To create a Public Access Link, go to the Form's :guilabel:`Public Access` tab. Click on :guilabel:`Create Public Access Link…` to begin.
+
+   .. image:: /img/central-submissions/public-link-new.png
+
+In the window that appears, you'll need to name the Link. This name is for your own identification purposes in the administration website, and is not displayed to respondents. You'll also need to decide whether to allow multiple submissions per respondent. Check the :guilabel:`Multiple responses` checkbox if you wish to allow it.
+
+.. admonition:: Single submission enforcement
+
+  In Enketo, the enforcement limiting each respondent to a single response is done with in-browser tracking. This means that a user could submit multiple times using different devices or browsers, or distribute the link beyond the intended group.
+
+  Also because of this tracking method, respondents will only be able to respond once *per Form*, not once per single-submission Link. Future versions of Central may change how this works. Please leave `feedback on the community forum <https://forum.getodk.org/c/features/9>`_ if this is something you'd like to see.
```

Comment:
Did you make an explicit decision not to mention that a respondent could actually submit again to the same form through a Link without enforcement? I can imagine users running into this and being confused or getting bad data.

An idea:
Also because of this tracking method, the enforcement is across all Single Submission Links for a given Form. That is, after making a Submission from a Link with single submission enforcement, a respondent won't be able to submit from any other Link with enforcement. However, they could submit again through a Link without enforcement.

Reply:
i think that makes me even more confused. let's just actually fix it?

Reply:
> let's just actually fix it?

What do you mean?

Reply:
right, i think we need to make central control the link. i think trying to document our way out of this is a fool's errand. it won't make sense to anybody ever.

Reply:
I might try to iterate on some wording in a follow-on PR but no reason to block.